deprecated_store/index.ts

```
@@ -0,0 +1,396 @@
+          } catch (error) {
+            if (error instanceof Error) {
+              console.error(`Load data failed while getting ${resource} due. Reason:`, error.message);
+            } else if (typeof error === "string") {
+              console.error(`Load data failed while getting ${resource}. Reason: ${error}`);
+            } else {
+              console.error(`Load data failed while getting ${resource} for unknown reason.`, error);
+            }
+          } finally {
+            get().endBatching();
+            return results;
+          }
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)